src/wp-includes/functions.php

```diff
@@ -4224,12 +4241,12 @@ function _xml_wp_die_handler( $message, $title = '', $args = array() ) {
 
 	$xml = <<<EOD
 <error>
-    <code>{$parsed_args['code']}</code>
```

Comment:
Looks like this change needs to be reverted?

Reply:
reverted